src/mockttp.ts

```diff
@@ -28,6 +28,7 @@ export interface Mockttp {
      * Stop the mock server and reset the rules.
      */
     stop(): Promise<void>;
+    pendingMocks(): Promise<any>;
```

Comment:
I'd prefer it if this method had a verb in the name. getPendingMocks()?

Reply:
Maybe method's name could be getPendingMocks. The name 'pendingMocks' is come from 'nock' because I just wanted a similar feature like pendingMocks() in nock (https://www.npmjs.com/package/nock#pendingmocks).